src/state/queries/notifications/util.ts

```diff
@@ -22,7 +22,7 @@ import {
   type NotificationType,
 } from './types'
 
-const GROUPABLE_REASONS = ['like', 'repost', 'follow']
+const GROUPABLE_REASONS = ['like', 'repost', 'follow', 'like-via-repost', 'repost-via-repost']
```

Comment:
```suggestion
const GROUPABLE_REASONS = [
  'like',
  'repost',
  'follow',
  'like-via-repost',
  'repost-via-repost',
]
```
I think there was a missing comma after `'repost-via-repost'` in your original commit

Reply:
Ahh thanks